recipes/ipychart/meta.yaml

```diff
@@ -0,0 +1,42 @@
+{% set name = "ipychart" %}
+{% set version = "0.1.3" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 0e188841f87379bf01105fd6b5d4649a8183d6de2be2c8b7677ea2b7d750ce7c
+
+build:
+  number: 0
+  noarch: python
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - python
+    - pip
+  run:
+    - python
+    - ipywidgets >=7.0.0
```

Comment:
It looks like there is also dependencies on a few other packages, that are now covered in:

https://github.com/nicohlr/ipychart/blob/master/requirements.txt

These will need to be added.